### PR TITLE
[ao] disabling dynamic conv/convT ops

### PR DESCRIPTION
**Pull request by HDCharles**
Summary: as mentioned in https://github.com/pytorch/pytorch/issues/70480 the dynamic conv ops are currently missing a key feature to bring their performance in line with other dynamic ops, this diff disables conv/convT from being automatically quantized with convert dynamic

Test Plan: buck test //caffe2/test:quantization --test-selectors test_quantized_module#TestDynamicQuantizedModule

Differential Revision: D33511152

### CODE REVIEW

File: torch/ao/quantization/quantization_mappings.py [hunk 1]
```diff
@@ -112,12 +112,13 @@
     nni.LinearReLU: nniqd.LinearReLU,
     nn.EmbeddingBag: nnq.EmbeddingBag,
     nn.Embedding: nnq.Embedding,
-    nn.Conv1d: nnqd.Conv1d,
-    nn.Conv2d: nnqd.Conv2d,
-    nn.Conv3d: nnqd.Conv3d,
-    nn.ConvTranspose1d: nnqd.ConvTranspose1d,
-    nn.ConvTranspose2d: nnqd.ConvTranspose2d,
-    nn.ConvTranspose3d: nnqd.ConvTranspose3d,
+    # Don't want to enable these by default because the performance is poor compared to other dynamic ops
```
Comment (vkuzo) [inline torch/ao/quantization/quantization_mappings.py:115]: nit: do we mean "accuracy"?  performance is usually used for "inference time" in this context

**Comment by vkuzo** [comment]
should we also add a warning to the constructors of the Python modules?  We can remove it once the more accurate kernels are there.

File: torch/ao/quantization/quantization_mappings.py (hunk 1 above)
Comment (HDCharles) [reply torch/ao/quantization/quantization_mappings.py:115]: yeah, i'll fix that

**Review by vkuzo — APPROVED** [review]
